docs/advanced.md

```diff
@@ -52,6 +52,21 @@ finally:
     client.close()
 ```
 
+There is some overhead to managing these connections, so you should try to persist your client as much as possible.
```

Comment:
> There is some overhead to managing these connections, so you should try to persist your client as much as possible.

👍 Yup.

I wonder if we might add something like "This can be achieved either by having a single scoped client that's passed throughout wherever it's needed, or by having a single global client instance."